Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -4,6 +4,8 @@ import Foundation
 import Nimble
 import XCTest
 
+public let LEGACY_HELLO = "ismaster"
```

Comment:
because hello was backported all the way to 3.6 and we test against relatively recent patch versions on Evergreen, we are safe to just always use "hello" when we need to run the command manually in tests to figure out topology info.

however, we still need this constant in some cases, as pre-4.2 servers will continue to use legacy hello for monitoring since helloOk was not backported to those versions.